test/Microsoft.Web.LibraryManager.Vsix.Test/UI/Models/LibraryIdViewModelTests.cs

```diff
@@ -41,7 +44,7 @@ public async Task FilterCompletions_SearchTextDoesNotContainAtSign_ReturnAllMatc
         {
             var testObj = new LibraryIdViewModel(GetTestSearchService(), "test");
 
-            CompletionSet result = await testObj.GetCompletionSetAsync(caretIndex: 0);
+            CompletionSet result = await testObj.GetCompletionSetAsync(caretIndex: 4);
```

Comment:
Why do these need to change?  When I wrote the tests originally, the caretIndex was actually kind of meaningless on this code path, which is why they were set to 0.